### PR TITLE
[DOC-1979] Add FG BIND_FETCH_FILES_TASK_TO_DELEGATE ff

### PR DESCRIPTION
**Pull request by brian-f-harness**
For reviewers, a preview is available.

https://65568f70ad7e400ea4afa51f--harness-developer.netlify.app/docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests#option-bind-tasks-to-the-delegate

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md [hunk 1]
```diff
@@ -89,6 +89,12 @@ You can use your Git repo for the configuration files in **Manifests** and Har
 
 Remote files can also use Go templating.
 
+### Option: Bind tasks to the delegate
+
+To ensure the Kubernetes in-cluster delegate is always auto assigned to perform tasks related to the Kubernetes deployment, for example fetching manifests or `values.yaml` remote files, you can bind the tasks to the delegate.
```
Comment (michaelcretzman) [inline docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md:94]:
```suggestion
To ensure the Kubernetes in-cluster delegate is always automatically assigned to perform tasks related to the Kubernetes deployment, for example fetching remote manifests or `values.yaml` files, you can bind the tasks to the delegate.
```

File: docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md [hunk 2]
```diff
@@ -89,6 +89,12 @@ You can use your Git repo for the configuration files in **Manifests** and Har
...
+
+This feature is behind the feature flag `BIND_FETCH_FILES_TASK_TO_DELEGATE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature flag.
```
Comment (michaelcretzman) [inline docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md:96]: typically we put this in a note (see any note in any topic markdown for an example) and directly under the heading.

**Review by michaelcretzman — APPROVED** [review]
approve with suggestions

File: docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md:96]: Updated, thanks

File: docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/first-gen/continuous-delivery/kubernetes-deployments/define-kubernetes-manifests.md:94]: updated

**Pull request merged**